lib/SQL/Abstract/mysql.pm

```diff
@@ -1,8 +1,20 @@
 package SQL::Abstract::mysql;
 use Mojo::Base 'SQL::Abstract';
 
+use Mojo::JSON 'encode_json';
+
 BEGIN { *puke = \&SQL::Abstract::puke }
 
+sub new {
+  my $self = shift->SUPER::new(@_);
+
+  # -e and -ne op
+  push @{$self->{unary_ops}}, {regex => qr/^e$/, handler => '_where_op_EXISTS',},
+    {regex => qr/^ne$/, handler => '_where_op_NOT_EXISTS',};
```

Comment:
If you want to be "git blamed" for these lines, then you have to remove the extra comma before `}`.